### PR TITLE
New ScrollablePanel component

### PR DESCRIPTION
**Pull request by aprentout**
### What does this PR do?

Related to: #[VEL-3967](https://linear.app/upfluence/issue/VEL-3967/create-ossscrollablepanel-which-handles-top-and-bottom-shadows)

### What are the observable changes?

🇮🇹 utilizzare in un contenitore

https://github.com/user-attachments/assets/0036ea7b-9797-4242-9806-47a0270f08e6

🇮🇹 utilizzare in un finestra modale

https://github.com/user-attachments/assets/ae694250-fb2b-462a-a35e-137e7c5ff8fd

🇮🇹 utilizzare in un pannello laterale

https://github.com/user-attachments/assets/2c4e639b-3494-4982-bdc8-b2b1b526bfb5

### Good PR checklist
- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [x] Added/updated tests
- [x] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: tests/dummy/app/controllers/application.js
Comment (aprentout) [inline tests/dummy/app/controllers/application.js:1]: Most of the file has been moved to dummy index controller

File: tests/dummy/app/templates/application.hbs
Comment (aprentout) [inline tests/dummy/app/templates/application.hbs:1]: Keeped only the sidebar and add an {{outlet}}

Comment (aprentout) [reply tests/dummy/app/templates/application.hbs:1]: Existing component have been moved to index :)

**Review by JulienVannier66 — APPROVED** [review]

File: addon/components/o-s-s/scrollable-panel.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+export default class OSSScrollablePanelComponent extends Component<OSSScrollablePanelComponentSignature> {
+  @tracked declare parentElement: HTMLElement;
+  @tracked shadowTopVisible: boolean = false;
+  @tracked shadowBottomVisible: boolean = false;
+
+  @action
+  initScrollListener(element: HTMLElement): void {
+    this.parentElement = element;
+    this.displayBottomShadow();
+    element.addEventListener('scroll', () => {
```
Comment (Miexil) [inline addon/components/o-s-s/scrollable-panel.ts:16]: cleanup with removeEventListener if possible

**Review by Miexil — APPROVED** [review]
Other than that, we can check if this works in exiting contexts 😇

**Review by OwenCoogan — APPROVED** [review]

**Pull request merged**